front-end-project/src/App.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import { route } from 'react-router-dom';
+
+import './App.css';
+import CreateNote from './components/CreateNote/CreateNote';
+import Nav from './components/Navigation/Nav';
+import Notes from './components/Notes/Notes';
+
+class App extends Component {
```

Comment:
Make sure to include a componentDidMount() with a get call to populate your state with all the notes already in the database.